resources/maps/korea/manifest.json

```
@@ -0,0 +1,159 @@
+{
+  "map": {
+    "height": 1670,
+    "num_land_tiles": 1179770,
+    "width": 1800
+  },
+  "mini_map": {
+    "height": 835,
+    "num_land_tiles": 290434,
+    "width": 900
+  },
+  "name": "Korea",
+  "nations": [
+    {
+      "coordinates": [1137, 849],
+      "flag": "kr",
+      "name": "Seoul",
+      "strength": 2
+    },
+    {
+      "coordinates": [1427, 983],
+      "flag": "kr",
+      "name": "South Korea",
+      "strength": 2
+    },
+    {
+      "coordinates": [974, 437],
+      "flag": "kp",
+      "name": "Pyongyang",
+      "strength": 2
+    },
+    {
+      "coordinates": [1123, 598],
+      "flag": "kp",
+      "name": "Democratic People's Republic of Korea",
```

The char limit before the name cuts off is 27, which causes issues with the Democratic Republic of the Congo. I think this might run into the same issue.
```suggestion
      "name": "North Korea",
```